modules/imgcodecs/test/test_png.cpp

```diff
@@ -106,6 +106,56 @@ TEST(Imgcodecs_Png, read_color_palette_with_alpha)
     EXPECT_EQ(img.at<Vec3b>(0, 1), Vec3b(0, 0, 255));
 }
 
+TEST(Imgcodecs_Png, load_save_multiframes_rgba)
+{
+    const string root = cvtest::TS::ptr()->get_data_path();
+    const string filename = root + "readwrite/OpenCV_logo_white.png";
+    vector<Mat> png_frames;
+
+    Mat image = imread(filename, IMREAD_UNCHANGED);
+    png_frames.push_back(image.clone());
+    Mat roi = image(Rect(0, 680, 680, 220));
+
+    for (int i = 0; i < 15; i++)
+    {
+        roi = roi - Scalar(0, 0, 0, 20);
+        png_frames.push_back(image.clone());
+    }
+
+    string output = cv::tempfile(".png");
+    EXPECT_EQ(true, imwrite(output, png_frames));
+    vector<Mat> read_frames;
+    EXPECT_EQ(true, imreadmulti(output, read_frames, IMREAD_UNCHANGED));
+    //EXPECT_EQ(png_frames.size(), read_frames.size());
```

Comment:
This PR will be working once this line is uncommented right?

Reply:
yes